yarn.lock

```diff
@@ -2753,6 +2753,11 @@ ansi-regex@^4.0.0, ansi-regex@^4.1.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-4.1.0.tgz#8b9f8f08cf1acb843756a839ca8c7e3168c51997"
   integrity sha512-1apePfXM1UOSqw0o9IiFAovVz9M5S1Dg+4TrDwfMewQ6p/rmMueb7tWZjQ1rx4Loy1ArBggoqGpfqqdI4rondg==
 
+ansi-regex@^5.0.0:
```

Comment:
all of these changes are coming from adding `source-map-explorer`?

Reply:
I hope so. Can I check it by uninstalling `source-map-explorer` and see if all diff are gone?

Reply:
to check this you have to restore `yarn.lock` from master and run `yarn add source-map-explorer`

Reply:
Looks like installing `source-map-explorer` adds all of it.